hudi-client/src/test/java/org/apache/hudi/client/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -80,9 +83,22 @@
 import static org.mockito.Mockito.when;
 
 @SuppressWarnings("unchecked")
+@RunWith(Parameterized.class)
 public class TestHoodieClientOnCopyOnWriteStorage extends TestHoodieClientBase {
 
   private static final Logger LOG = LogManager.getLogger(TestHoodieClientOnCopyOnWriteStorage.class);
+  private final IndexType indexType;
+
+  @Parameterized.Parameters(name = "{index}: Test with IndexType={0}")
+  public static Collection<Object[]> data() {
+    Object[][] data =
+        new Object[][] {{IndexType.BLOOM}};
```

Comment:
@vinothchandar : have removed other index types to see travis CI succeed. I got 
`
The job exceeded the maximum time limit for jobs, and has been terminated.` 
in travis CI when enabled.